ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Mempool.hs

```diff
@@ -39,6 +41,8 @@ import           Ouroboros.Consensus.Shelley.Ledger.Block
 import           Ouroboros.Consensus.Shelley.Ledger.Ledger
 import           Ouroboros.Consensus.Shelley.Protocol
 
+import           Ouroboros.Network.Block (unwrapCBORinCBOR, wrapCBORinCBOR)
```

Comment:
Nitpick: can this import be moved between the `Cardano.*` and `Ouroboros.Consensus.*` imports (separated by blank lines)?